helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json

```diff
@@ -1742,7 +1760,7 @@
         },
         "datasource": {
```

Comment:
Thé whole thing:

"datasource": {
          "type": "prometheus",
          "uid": "${datasource}"
        },

Can bé replaced with "datasource": "$datasource", but it's not blocking

Reply:
I'm keeping this as is